UI/NpcList/NPCRow.gd

```diff
@@ -0,0 +1,54 @@
+extends PanelContainer
+
+onready var _npcNameLabel = $HBoxContainer/Name
+onready var _npcGenderLabel = $HBoxContainer/Gender
+onready var _npcPersonalityLabel = $HBoxContainer/Personality
+onready var _amountOfChildrenLabel = $HBoxContainer/Children
+onready var meetButton = $HBoxContainer/CenterContainer2/Meet
+onready var forgetButton = $HBoxContainer/CenterContainer/Forget
+onready var showNpc = $ShowNPC
+var _npcID: String
+
+
+func setName(name: String):
+	_npcNameLabel.text = name
+
+
+func setGender(gender: String):
+	_npcGenderLabel.text = gender
+
+
+func setPersonality(personality: String):
+	_npcPersonalityLabel.text = personality
+
+
+func setChildren(children):
+	_amountOfChildrenLabel.text = String(children)
+
+
+func setNpcID(ID: String):
+	if(_npcID == null):
```

Comment:
this probably should be `if(ID == null):` right?
But when you specify the type of the argument Godot actually ensures that it won't be null so you don't really need that check. You would need it if your function declaration looked like `func setNpcID(ID):` (without the : String part)